src/components/Form/FormFields/Utils.ts

```diff
@@ -32,6 +32,7 @@ export type FormFieldBaseProps<T> = {
   error?: FieldError;
   onFocus?: (event: FocusEvent<HTMLInputElement>) => void;
   onBlur?: (event: FocusEvent<HTMLInputElement>) => void;
+  help?: boolean;
```

Comment:
What does help mean? Did you mean showHelp? 

Let's make sure variable naming is always good.

Reply:
Not done 🙄